### ISSUE TITLE
1D CNN

### ISSUE THREAD

**Issue by sum1lim** [sum1lim/sea_ice_remote_sensing#20]
GLCM features do not work well with the regular neural network and degrade the performance of the model. It is suspected that the addition of the features makes the network too complicated, which prevents finding the global minimum of the loss function. Write a script for a 1D convolutional neural network as a potential solution.
